libraries/GCS_MAVLink/GCS.h

```diff
@@ -183,6 +183,9 @@ class GCS_MAVLINK
 
     // mission item index to be sent on queued msg, delayed or not
     uint16_t mission_item_reached_index = AP_MISSION_CMD_INDEX_NONE;
+	
+    // frsky backend
+    AP_Frsky_Telem frsky;
```

Comment:
This adds an extra instance of Frsky telem per GCS backend, which is almost definitely not what you want to do.

Reply:
The AP_Frsky_Telem was not declared int the GCS_MAVLink class.  It is in the GCS class, which is where it was that line was copied from.  Are you suggesting moving the declaration to the top of the GCS.h file?  To make it generic?

Reply:
I changed the approach to eliminate the library. 

Thank you!